src/test/unit/components/member-card/MembersCardPresentation.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { renderWithProviders } from "../../../../test__utils/renderWithProvides";
+import { membersData } from "../../../../mocks/db/members";
+
+import MembersCardPresentation from "../../../../components/MembersSectionNew/components/MembersCard/Presentation";
+import SettingButton from "../../../../components/SettingButton/SettingButton";
+
+describe("MembersCardPresentation", () => {
+  test("should renders correctly", () => {
```

Comment:
please write the test case3 message properly, I don't understand here what are you testing

Reply:
okay